src/main/java/edu/wpi/grip/ui/preview/PreviewsView.java

```diff
@@ -28,12 +33,14 @@
 
     private final EventBus eventBus;
     private final List<OutputSocket<?>> previewedSockets;
+    private PipelineView pipeline;//This is used to determine the order the previews are shown in.
```

Comment:
Is this ever set anywhere other than the constructor? If not then you can make it final.